File: conditionals/time.py
# David Garcia Time Py
import time

#first instance of time in proramming
first_time = time.gmtime()
#print(first_time)

#Current time in seconds 
current = time.time()
#print(current) #seconds since Jan 1 1970

#Current date and time like we see it normally 
now = time.ctime(current)
#print(now)

#get just a part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour
print(day)
print(hour)
print(local_time) #military time

if hour <= 12:
    print("Good morning")

File: conditionals/time.c
// David Garcia notes time c
#include <stdio.h>
#include <time.h>

int main(void){
    // Time since Jan 1 1970
    time_t seconds;

    seconds = time(NULL);

    //printf("Seconds since Jan 1 1970 = %d\n", seconds);

    //Current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    //printf("current time and date is %s\n", asctime(timeinfo));
    
    //Current hour
    time_t now = time(NULL);

    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;  // military time
    printf("%d\n", hour);

    if(hour <12);
    printf("Good morning");
    return 0;
}